README.md

```diff
@@ -19,6 +19,7 @@ hijacking secure communications without getting caught.
 ## Golang Library
 The packages in this library implement the various components of the CONIKS system and may be imported individually.
 
+- ``client``: Client-side protocol implementation
```

Comment:
Should this explanation say something more like "Client-side protocol cgo (or C) hooks"?